packages/backpack-core/config/webpack.config.js

```diff
@@ -115,7 +115,14 @@ module.exports = (options) => {
       new webpack.BannerPlugin({
         raw: true,
         entryOnly: false,
-        banner: `require('${require.resolve('source-map-support/register')}')`
+        banner: `require('${
+            // Is source-map-support installed as project dependency, or linked?
+            require.resolve('source-map-support').match(process.cwd())
```

Comment:
`.indexOf(process.cwd()) === 0` would be more reliable than `.match(process.cwd())` if `.match` here is `String.prototype.match`, not anything special.

Reply:
This banner is writing absolute paths on bundles, and it's hard to notice unless you are using some `CI` environment

Reply:
created a issue here https://github.com/jaredpalmer/backpack/issues/124